go-controller/pkg/util/node_annotations.go

```diff
@@ -63,6 +63,14 @@ const (
 	// OvnNodeManagementPortMacAddress is the constant string representing the annotation key
 	OvnNodeManagementPortMacAddress = "k8s.ovn.org/node-mgmt-port-mac-address"
 
+	// OvnNodeManagementPortMacAddresses contains all mac addresses of the management ports
+	// on all networks keyed by the network-name
+	// k8s.ovn.org/node-mgmt-port-mac-addresses: {
+	// "default":"ca:53:88:23:bc:98",
+	// "l2-network":"5e:52:2a:c0:98:f4",
+	// "l3-network":"1a:2c:34:29:b7:be"}
+	OvnNodeManagementPortMacAddresses = "k8s.ovn.org/node-mgmt-port-mac-addresses"
```

Comment:
If I remember correctly the mgmt port IP address is different per network, it would be possible to generate the mac from there so we don't need this annotation ?

Reply:
unfortunately no, the MAC address is not coming from the IP and also for L2 its actually same .2 on each node of same network; so its usually generated on the node side and then used by OVN networks...